packages/headless/src/controllers/core/facets/category-facet/headless-core-category-facet.test.ts

```diff
@@ -295,7 +295,7 @@ describe('category facet', () => {
         expect(categoryFacet.state.canShowMoreValues).toBe(true);
       });
 
-      it('if #moreValuesAvailable is true, #state.canShowMore is true', () => {
+      it('if #moreValuesAvailable is false, #state.canShowMore is false', () => {
```

Comment:
should add more tests IMO

Reply:
Entirely agree 💯
I should have put that test are Todo in the pr desc